developer/src/kmc-model/package.json

```diff
@@ -16,8 +16,7 @@
     "/build/src/"
   ],
   "scripts": {
-    "build": "tsc -b && npm run build-cjs",
-    "build-cjs": "esbuild build/src/lexical-model-compiler.js --bundle --platform=node --external:../../node_modules/* > build/cjs-src/lexical-model-compiler.cjs",
+    "build": "tsc -b",
```

Comment:
One extra throw-in.  Since Web is now es-module based on this branch, this package doesn't need to export a .cjs version anymore.

Reply:
OK. Is this the only place where cjs-src is referenced?

Reply:
I double-checked; it is.

`kmc` does build a bundle, but it's of `kmc`'s wrapper-action that utilizes this package; it doesn't need this cjs either.